run_prestage.py

```diff
@@ -148,132 +151,105 @@ def get_Xcache_filelist(fnames: list):
                 for sample_name in data_l:
                     if data_letter in sample_name:
                         new_sample_list.append(sample_name)
         # take bkg
+        bkg_l = [sample_name for sample_name in dataset.keys() if "data" not in sample_name]
+        logger.info(f"background samples: {bkg_l}")
         bkg_samples = args.bkg_samples
-        print(f"bkg_samples: {bkg_samples}")
+        logger.info(f"bkg_samples: {bkg_samples}")
         if len(bkg_samples) >0:
-            for bkg_sample in bkg_samples:
-                if bkg_sample.upper() == "DY": # enforce upper case to prevent confusion
-                    # new_sample_list.append("dy_M-50")
-                    new_sample_list.append("dy_M-100To200")
-                    # new_sample_list.append("dy_VBF_filter")
-                    # new_sample_list.append("dy_m105_160_vbf_amc")
-                    # new_sample_list.append("dy_VBF_filter_customJMEoff")
-                    # new_sample_list.append("dy_M-50To120")
-                    # new_sample_list.append("dy_M-120To200")
-                    # new_sample_list.append("dy_VBF_filter_Amandeep10_6_26")
-                    # new_sample_list.append("dy_VBF_filter_Amandeep10_6_32")
-                    # new_sample_list.append("dy_VBF_filter_Amandeep_fromGenSim")
-                    # new_sample_list.append("dy_VBF_filter_fromGridpack")
-                    
-                    # Run3 specific datasets:
-                    # new_sample_list.append("dy_M-50To120")
-                    # new_sample_list.append("dy_M-120To200")
-                
+            for bkg_letter in bkg_samples:
+                for bkg_name in bkg_l:
+                    if bkg_letter in bkg_name:
+                        new_sample_list.append(bkg_name)
+
+        logger.debug(f"new sample list: {new_sample_list}")
 
-                
-                elif bkg_sample.upper() == "TT": # enforce upper case to prevent confusion
-                    new_sample_list.append("ttjets_dl")
-                    new_sample_list.append("ttjets_sl")
-                elif bkg_sample.upper() == "ST": # enforce upper case to prevent confusion
-                    new_sample_list.append("st_tw_top")
-                    new_sample_list.append("st_tw_antitop")
-                elif bkg_sample.upper() == "VV": # enforce upper case to prevent confusion
-                    new_sample_list.append("ww_2l2nu")
-                    new_sample_list.append("wz_3lnu")
-                    new_sample_list.append("wz_2l2q")
-                    new_sample_list.append("wz_1l1nu2q")
-                    new_sample_list.append("zz")
-                elif bkg_sample.upper() == "EWK": # enforce upper case to prevent confusion
-                    new_sample_list.append("ewk_lljj_mll50_mjj120")
-                    # new_sample_list.append("ewk_lljj_mll105_160_ptj0")
-                    # new_sample_list.append("ewk_lljj_mll105_160_py_dipole")
```

Comment:
@green-cabbage Do we really need these lines?

Reply:
hmmm, those are other background MC samples that we keep locally in Purdue. They're ttbar, single top, diboson and electroweak Z production samples, so I would argue that we need them